packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -212,7 +212,9 @@ function WorklistDataGrid(props) {
   const [focusedCol, setFocusedCol] = useState(0);
   const [ariaLiveMessage, setAriaLiveMessage] = useState(null);
   const [cellAriaLiveMessage, setCellAriaLiveMessage] = useState(null);
-  const rowSelection = useRef({ isRowSelectionModeToggledByGrid: false });
+  const isRowSelectionModeToggledByGrid = useRef(false);
+  const inShiftUpDownMode = useRef(false);
```

Comment:
I cannot understand the purpose of this variable based on its name.  What is the shift up/down mode?  Could there be a more clear name that can be used?

Reply:
We will accept suggestions for this variable name. When the user starts to select a range of rows using the Shift+Up or the Shift+Down, it establishes a new anchor. As long as the user does not release the shift key, another Shift+Up/Down continues the selection or unselection using the same anchor established by the initial Shift+Up/Down. We needed a variable to distinguish between the first Shift+Up/Down and subsequent ones.